src/surge-xt/gui/overlays/LuaEditors.cpp

```diff
@@ -45,8 +45,9 @@ namespace Overlays
     TextfieldPopup
     Base class that can be used for creating other textfield popups like the search
 */
-TextfieldButton::TextfieldButton(juce::String &svg) : juce::Component()
+TextfieldButton::TextfieldButton(juce::String &svg, int r) : juce::Component()
```

Comment:
I would name this second argument more nicely so that it it's clearer, rather than guesswork.